drivers/rilmodem/rilutil.h

```diff
@@ -76,6 +76,11 @@ struct ril_gprs_context_data {
 	enum ofono_gprs_context_type type;
 };
 
+struct ril_voicecall_driver_data {
+	GRil *gril;
```

Comment:
The same name is used in the other two structures for passing info to the atom drivers: see ril_sim_data and ril_gprs_context_data in the same file. Probably we should change all names to "ril", but we can leave that for a future refactoring PR.